scripts/java/BuildAndRun.java

```diff
@@ -0,0 +1,105 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Base64;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Stream;
+
+public class BuildAndRun {
+    public static void main(String[] args) throws IOException, InterruptedException {
```

Comment:
I would expect to see a bit of logging in this code so it would be easier to track failures.